src/elements/kano-tracking/kano-tracking-behavior.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../../js/config.html">
+<script>
+    (function (Kano) {
+
+        window.dataLayer = window.dataLayer || [];
+
+        // @polymerBehavior
+        Kano.TrackingBehavior = {
+            /**
+             * Check whether the user is using a Kano Kit
+             */
+            checkUserDevice () {
+                let type = this.isPi() ? 'paid' : 'free';
+                this.dispatchTrackingEvent({
+                    userType: type
+                });
+            },
+            /**
+             * Wrapper for dispatching events to GTM datalayer
+             *
+             * @param {Object || String} trackingEvent
+             */
+            dispatchTrackingEvent (trackingEvent) {
+                let payload = trackingEvent;
+                if (trackingEvent !== Object(trackingEvent)) {
+                    payload = {
+                        event: trackingEvent
+                    };
+                }
+                window.dataLayer.push(payload);
+            },
+            /**
+             * Generic function to check whether the user is using a Kano Kit
+             *
+             * note: not currently fullproof
+             */
+            isPi() {
+                let userAgent = window.navigator.userAgent;
```

Comment:
Good with that. For further reference here are the userAgents on the kit: `Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.7.1 Chrome/49.0.2623.111 Safari/537.36`, `Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.91 Safari/537.36` and `Mozilla/5.0 (X11; Linux armv7l) AppleWebKit/601.1 (KHTML, like Gecko) Version/8.0 Safari/601.1` and their equivalent with `armv6l` is the regexp matches these it should cover most cases. The risk is that phones have arm chips too